src/app/modules/registration/components/edit-images/edit-images.component.html

```diff
@@ -51,6 +87,43 @@
       [rows]="2"
       placeholder="{{ pictureCommentPlaceholder() | translate }}"
     ></app-text-comment>
+    <ion-accordion-group>
+      <ion-accordion>
+        <ion-item slot="header" class="accordion-button">
+          <ion-label>{{ "REGISTRATION.SNOW.SNOW_PROFILE.STRAT_PROFILE.SHOW_MORE" | translate }}</ion-label>
+        </ion-item>
+        <div class="ion-padding" slot="content">
+          <ion-item>
+            <app-select
+              label="REGISTRATION.IMAGE_ASPECT"
+              [(selectedValue)]="attachment.Aspect"
+              [options]="aspectOptions"
+              (ionChange)="addNewAttachmentAspect(attachment, $event)"
+            ></app-select>
+          </ion-item>
+          <ion-item>
+            <ion-input
+              [value]="attachment.Photographer"
+              (ionChange)="addNewAttachmentPhotographer(attachment, $event)"
+              label="PHOTOGRAPHER"
+              labelPlacement="stacked"
+              placeholder="Photographer"
+            >
+            </ion-input>
+          </ion-item>
+          <ion-item>
+            <ion-input
+              [value]="attachment.Copyright || userSettings()?.copyright"
```

Comment:
Hvis verken attachment.Copyright eller userSettings()?.copyright er satt, skal du vise kallenavnet.
Nå vises ingenting.

Reply:
da må vi først få kallenavn redigering til develop. Jeg endrer hvordan vi får tak i kallenavnt. Nesten umulig å få den i nåværende oppsetting.

Reply:
Fint. Nå virker det for redigering av bilde, men ikke for nytt bilde. Tar du nytt bilde også?